ui/app/adapters/clients/version-history.js

```diff
@@ -0,0 +1,13 @@
+import Application from '../application';
+
+export default Application.extend({
+  queryRecord() {
```

Comment:
I believe we get free caching from ember-data if we instead use the `findAll` method, which we can use here since we're not actually using any query params it's just a static API call.